From: Sascha Steinbiss <satta@debian.org>
Date: Mon, 23 Jan 2017 23:42:09 +0000
Subject: Enable full hardening

---
 stenotype/Makefile | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/stenotype/Makefile b/stenotype/Makefile
index 1f96db5..cee501f 100644
--- a/stenotype/Makefile
+++ b/stenotype/Makefile
@@ -53,13 +53,13 @@ FUZZ_FILES=util index index_bin
 SANITIZE=
 ifndef SANITIZE
 	OBJECTS=$(foreach file,$(FILES),$(file)_gcc.o)
-	CFLAGS=$(SHARED_CFLAGS) $(GCC_CFLAGS)
-	LDFLAGS=$(SHARED_LDFLAGS) $(GCC_LDFLAGS)
+	CFLAGS:=$(CFLAGS) $(SHARED_CFLAGS) $(GCC_CFLAGS)
+	LDFLAGS:=$(LDFLAGS) $(SHARED_LDFLAGS) $(GCC_LDFLAGS)
 	CC=$(GCC)
 else
 	OBJECTS=$(foreach file,$(FILES),$(file)_$(SANITIZE)_clang.o)
-	CFLAGS=$(SHARED_CFLAGS) $(CLANG_CFLAGS) -fsanitize=$(SANITIZE)
-	LDFLAGS=$(SHARED_LDFLAGS) $(CLANG_LDFLAGS)
+	CFLAGS:=$(CFLAGS) $(SHARED_CFLAGS) $(CLANG_CFLAGS) -fsanitize=$(SANITIZE)
+	LDFLAGS:=$(LDFLAGS) $(SHARED_LDFLAGS) $(CLANG_LDFLAGS)
 	CC=$(CLANG)
 endif
 ifeq "$(SANITIZE)" "memory"
@@ -77,15 +77,15 @@ clean:
 
 # Generate g++ object files.
 %_gcc.o: %.cc $(DEPS)
-	$(GCC) $(CFLAGS) -c -o $@ $<
+	$(GCC) $(CFLAGS) $(CPPFLAGS) $(SHARED_CFLAGS) -c -o $@ $<
 
 # Generate clang object files.
 %_$(SANITIZE)_clang.o: %.cc $(DEPS)
-	$(CLANG) $(CFLAGS) -c -o $@ $<
+	$(CLANG) $(CFLAGS) $(CPPFLAGS) $(SHARED_CFLAGS) -c -o $@ $<
 
 # Generate the stenotype binary itself.  You mostly want this :)
 stenotype: $(OBJECTS)
-	$(CC) $(CFLAGS) -o $@ $^ $(LDFLAGS)
+	$(CC) $(CFLAGS) $(CPPFLAGS) -o $@ $^ $(LDFLAGS)
 
 
 
