From: Sascha Steinbiss <satta@debian.org>
Date: Mon, 23 Jan 2017 23:42:09 +0000
Subject: Fix spelling

---
 stenotype/stenotype.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/stenotype/stenotype.cc
+++ b/stenotype/stenotype.cc
@@ -250,7 +250,7 @@
     if (flag_gid == "") {
       flag_gid = "nogroup";
     }
-    LOG(INFO) << "Dropping priviledges from " << getgid() << " to GID "
+    LOG(INFO) << "Dropping privileges from " << getgid() << " to GID "
               << flag_gid;
     auto group = getgrnam(flag_gid.c_str());
     CHECK(group != NULL) << "Unable to get info for group " << flag_gid;
@@ -262,7 +262,7 @@
     if (flag_uid == "") {
       flag_uid = "nobody";
     }
-    LOG(INFO) << "Dropping priviledges from " << getuid() << " to UID "
+    LOG(INFO) << "Dropping privileges from " << getuid() << " to UID "
               << flag_uid;
     auto passwd = getpwnam(flag_uid.c_str());
     CHECK(passwd != NULL) << "Unable to get info for user " << flag_uid;
